frontend/src/features/mapSideBar/SidebarContents/subforms/AssociatedLandReviewPage.tsx

```diff
@@ -79,11 +79,15 @@ const LinkButton = styled.span`
  * @param {IReviewProps} props {IReviewProps}
  */
 export const AssociatedLandReviewPage: React.FC<any> = (props: IReviewProps) => {
-  const defaultEditValues = {
-    identification: true,
-    usage: true,
-    valuation: true,
-  };
+  const formikProps = useFormikContext<any>();
+  const defaultEditValues = useMemo(
```

Comment:
put all the sections into edit mode if the form is invalid, so that the validation errors are displayed.